src/debugger/Debugger.js

```diff
@@ -129,23 +129,31 @@ export class DebugServer {
     switch (command) {
       case DebugMessage.BREAKPOINT_ADD_COMMAND:
         invariant(args.kind === "breakpoint");
-        this._breakpoints.addBreakpoint(args.filePath, args.line, args.column);
-        this._channel.sendBreakpointAcknowledge(DebugMessage.BREAKPOINT_ADD_ACKNOWLEDGE, requestID, args);
+        for (let bp of args.breakpoints) {
```

Comment:
I would suggest you change "BreakpointCollection" class to "BreakpointManager" and move all these breakpoint related handling logic into BreakpointManager.